testing/src/main/java/org/oppia/testing/screenshots/ScreenshotManager.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.testing.screenshots
+
+import android.app.Activity
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Environment
+import android.view.View
+import java.io.File
+import java.io.FileOutputStream
+
+const val OUTPUT_FOLDER_NAME = "OppiaScreenshots"
+
+/**A class that is used to take screenshots from some activity.*/
```

Comment:
Nit: `/** A class that is used to take screenshots for a UI. */`

Reply:
Done